### PR TITLE
Update svm4x

### PR DESCRIPTION
**Pull request by psachs**
Rename SVM41 to SVM4x and remove SVM40 repositories.

SVM4x evaluation kit covers evaluation of SGP40 and SGP41 and therefore SVM40 evaluation kit is discontinued.

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @psachs. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

---

By the way, I see you have changed the value of the `name` field in the `library.properties` metadata files:

- https://github.com/Sensirion/arduino-uart-svm4x/commit/17971a5ea86dc60cf0de6ef82563bd21afa91157
- https://github.com/Sensirion/arduino-i2c-svm4x/commit/401ac6accdde56a3ee791099e88e4c308ef9bdcc

You will also need to submit a request for the registered library names to be updated once you make new releases in the library repositories with those changes:

https://github.com/arduino/library-registry/blob/main/FAQ.md#how-can-i-change-a-librarys-name

**Review by MatteoPologruto — APPROVED** [review]
Removal: https://github.com/arduino/library-registry/pull/2993#pullrequestreview-1501312383
URLs change: https://github.com/arduino/library-registry/pull/2994#pullrequestreview-1501329281

**Pull request merged**
